runtime-image/j9-11-overrides.yaml

```diff
@@ -0,0 +1,43 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap73-openj9-11-runtime-openshift-rhel8"
+description: "The JBoss EAP 7.3 openjdk11 runtime image"
```

Comment:
Should this image description mention  OpenJ9?
Something like "The JBoss EAP 7.3 runtime image for OpenJDK with OpenJ9 11"

Reply:
Good catch, thanks @jmesnil. I have fixed the wording.

Reply:
@levivic, while we are at it, we should also override the description for the (builder) image overrides in `rel-j9-11-overrides.yaml` :)

Reply:
There wasn't a description for that file. But I just added. Pushed. Thx.